Bots/Python/Skills/CodeFirst/EchoSkillBot/config.py

```diff
@@ -9,7 +9,7 @@
 class DefaultConfig:
     """ Bot Configuration """
 
-    PORT = 39783
+    PORT = 37400
     APP_ID = os.environ.get("MicrosoftAppId", "TODO: Add here the App ID for the skill bot")
```

Comment:
Remove TODOs and just leave empty strings so it works without appid and password and without making changes